src/App.tsx

```diff
@@ -1,61 +1,130 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getNewPostId(posts: Todo[]) {
+  return Math.max(...posts.map(post => post.id)) + 1;
```

Comment:
There is some redundant logic, you can make random ID with library uuidv4, or just with new Date()

Reply:
![image](https://github.com/mate-academy/react_add-todo-form/assets/104771337/12078f6e-1e9c-405f-a8ad-eefc1ad6f39d)

Reply:
![image](https://github.com/mate-academy/react_add-todo-form/assets/104771337/481be189-96fc-4661-b161-0f933808b0eb)

Reply:
![image](https://github.com/mate-academy/react_add-todo-form/assets/104771337/ee390975-df00-4215-994e-84b4ae0a0798)